services/credential-server/src/agent.ts

```diff
@@ -39,18 +50,70 @@ class Agent {
   }
 
   async issueAcdcCredentialByAid(schemaSaid, aid, attribute) {
+    if (schemaSaid === Agent.LE_SCHEMA_SAID) {
+      return this.issueLeChainedCredential(aid);
```

Comment:
As far as I can tell, the LE credential has a `LEI` attribute too. This should come from the UI in the same way it does for the QVI credential.